scripts/jira_tickets/create_compara_release_JIRA_tickets.pl

```diff
@@ -76,7 +76,12 @@ sub main {
     die 'Aborted by user. Please rerun with correct parameters.' if ( $response ne 'y' );
 
     # Create JIRA tickets
-    my $subtask_keys = $jira_adaptor->create_tickets(-JSON_FILE => $tickets_json, -DRY_RUN => $dry_run, -DEFAULT_PRIORITY => 'Blocker');
+    my $subtask_keys = $jira_adaptor->create_tickets(
+        -JSON_FILE        => $tickets_json,
+        -DEFAULT_PRIORITY => 'Blocker',
+        -EPIC_LINK        => 'ENSCOMPARASW-3572',
```

Comment:
I would move `ENSCOMPARASW-3572` to a constant defined at the top of the file. Otherwise, I think it's good to go